api/src/models/JobMetadata.ts

```diff
@@ -58,31 +60,61 @@ class JobMetadata {
     }
 
     get derivativeStatus(): Record<string, unknown> {
-        // TODO: populate with real data
+        let lockfileExists: boolean = fs.existsSync(this.derivativeLockfile);
         const status = {
             expected: 10,
```

Comment:
The 10 here was a fake value we made up just to see something in the UI. This should actually get initialized to 0 now that we're doing work for real.